


<style>
table {
    width:100%;
}
</style>



# Accessing public data
TL;DR here are the links:

| Data | Links |
| ------ | ------------------ | 
| Reads (DNA/RNA) &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Reads <br>https://www.ncbi.nlm.nih.gov/sra/?term=all%5Bfilter%5D<br>link2 |
| Genome | Assemblies & annotations <br>https://www.ncbi.nlm.nih.gov/assembly/?term=all%5Bfilter%5D<br><br> Genes <br>https://www.ncbi.nlm.nih.gov/gene/?term=all%5Bfilter%5D<br><br>Taxonomy<br>https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi |
| Transcriptomics | Gene expression<br>https://www.ncbi.nlm.nih.gov/gds/?term=all%5Bfilter%5D<br>link2 |
| Variation | Sequence variants (SNVs/SNPs, Indels) <br>https://www.ncbi.nlm.nih.gov/clinvar/?term=all%5Bfilter%5D<br>link1<br><br>Structural variants<br>link1 |
| Proteomics | Protein sequences<br>link1<br><br>Conserved domains<br>link1<br><br>Protein structures<br>link1 |

<br><br>


## Introduction
Thanks to research becoming more open in the last decade, there is now a ***huge amount*** of freely available data online.  The push towards repeatable analysis and open access to data has had enormous impact on medical science, and will continue to do so in the future.

This document is a ***beginners guide*** for how to access all this data. 

In each section, the following is covered:
* Who uses the data? (what type of analysis / field)
* How to access (download or use)
* Format of the data 

<br>

## Contents
* [Summary of common data repositories](#Common-data-Repositories)
    * [NCBI](#NCBI)
    * [ENA](#ENA)
    * [DDBJ](#DDBJ)

* [Reads](#Reads)
    * [DNA](#DNA)
    * [RNA](#RNA)
    * [Epigenetics](#Epigenetics)

* [Genomes](#Genomes)
    * [Assemblies](#Assemblies)
    * [Annotations](#Annotations)
    * [Genes](#Genes)
    * [Taxonomy](#Taxonomy)

* [Transcriptomics](#Transcriptomics)
    * [Expression data](#Expression-data)
    * [Transcript isoforms](#Transcript-isoforms)
    * [Networks & pathways](#Networks-&-pathways)

* [Variation](#Variation)
    * [Sequence variation (SNVs/SNPs, Indels)](#Sequence-variation-(SNVs/SNPs,-Indels))
    * [Structural variation](#Structural-variation)

* [Proteomics](#Proteomics)
    * [Protein structures](#Protein-structures)
    * [Protein interactions](#Protein-interactions)
    * [Protein expression](#Protein-expression)
    * [Protein domains and functions](#Protein-domains-and-functions)

<br><br>

## Common data Repositories
Jump to
* [NCBI](#NCBI)
* [ENA](#ENA)
* [DDBJ](#DDBJ)

<br>

### NCBI
https://www.ncbi.nlm.nih.gov/home/documentation/

Here are the commonly used databases within NCBI:
* **[Assembly](https://www.ncbi.nlm.nih.gov/assembly/?term=all%5Bfilter%5D)**<br>
Genome assemblies (contig, scaffold, draft and complete)

* **BioProject**<br>


* **BioSample**<br>


* **ClinVar**<br>
Genomic variation (all types - SNV/SNP, Indel, structural variants
etc)

* **dbVar**<br>
Structural variation (including clinical verdicts)

* **Gene**<br>
Links gene IDs with other data (name, symbol, publication, GO terms, medical variants etc)

* **GEO datasets**<br>
Gene expression - both from arrays, and sequence data

* **OMIM**<br>


* **Nucleotide**<br>
Large collection of nucleic acid sequences (incl. genes, proteins, genomes)

* **Protein**<br>
Protein sequences

* **SRA**<br>
Sequence read sets

* **Taxonomy**<br>
Taxonomic heirarchy of all known organisms

<br>

These resources are all described in sections below.

<br>

### ENA
https://www.ebi.ac.uk/ena/browser/about/content
* **BRENDA**<br>
Enzymes & enzyme-ligand interactions

* **ENA** <br>
Nucleotide sequence information - reads, assemblies, annotations

* **UniProt**<br>
Protein sequences and annotations

* **Human Protein Atlas**<br>
Expression & localisation of proteins

* **PDBe**<br>
Macromolecular structures (mainly proteins)

* **PRIDE**<br>
Mass-spectroscopy data

* **Reactome**<br>
Chemical pathways

* **SILVA**<br>
Ribosomal RNA sequences (16S, 23S etc)

* **STRING**<br>
Protein-protein interactions

* **UniProt**<br>
Protein sequences & annotations (comprehensive)

<br>

These resources are all described in sections below.

<br>

### DDBJ



<br><br>




## Reads
Jump to
* [DNA](#DNA)
* [RNA](#RNA)
* [Epigenetics](#Epigenetics)

<br>

### DNA

**Who uses the data?**

**How to access**
* SRA
* ENA
**Format**

<br>

### RNA
**Who uses the data?**

**How to access**

**Format**

<br>

### Epigenetics

**Who uses the data?**

**How to access**

**Format**

<br>





<br><br>

## Genomes
Jump to
* [Assemblies](#Assemblies)
* [Annotations](#Annotations)
* [Genes](#Genes)
* [Taxonomy](#Taxonomy)



### Assemblies
* NCBI Assembly
    * GenBank vs RefSeq 
* ENA
<br>

### Annotations
* ENCODE 
<br>

### Genes
* ENA
* SILVA


<br>

### Taxonomy

<br>





<br><br>

## Transcriptomics
Jump to
* [Expression data](#Expression-data)
* [Transcript isoforms](#Transcript-isoforms)
* [Networks & pathways](#Networks-&-pathways)

### Expression data
* GEO
* GTEx

<br>

### Transcript isoforms

<br>


### Networks & pathways
* REACTOME
* KEGG






<br><br>

## Variation
Jump to
* [Sequence variation (SNVs/SNPs, Indels)](#Sequence-variation-(SNVs/SNPs,-Indels))
* [Structural variation](#Structural-variation)

### Sequence variation (SNVs/SNPs, Indels)
* ClinVar
* OMIM
* GeneCards

<br>

### Structural variation






<br><br>

## Proteomics
jump to
* [Protein structures](#Protein-structures)
* [Protein interactions](#Protein-interactions)
* [Protein expression](#Protein-expression)
* [Protein domains and functions](#Protein-domains-and-functions)



### Protein structures
* PDB 
* PDBe 
<br>

### Protein interactions
* STRING
* BRENDA

<br>

### Protein expression
* PRIDE (mass spec data)
* Human Protein Atlas
<br>

### Protein domains and functions
* InterPro

<br><br>

